site/pages/create-disruption.tsx

```diff
@@ -1,67 +1,182 @@
-import { ReactElement } from "react";
-import { DisruptionsDateTimeInfo } from "../components/DisruptionDateTimeInfo";
+import React, { ReactElement } from "react";
+import { DisruptionInfo } from "interfaces";
+import { DisruptionsDateTimeInfo } from "components/DisruptionDateTimeInfo";
+import { MiscellaneousReason, PersonnelReason, EnvironmentReason, EquipmentReason } from "../../shared-ts/siriTypes";
 import { BaseLayout } from "../components/layout/Layout";
-import { DisruptionInfo } from "../interfaces";
+import { DISRUPTION_REASONS } from "../constants/index";
 
+DISRUPTION_REASONS;
 const title = "Create Disruptions";
 const description = "Create Disruptions page for the Create Transport Disruptions Service";
 
 interface CreateDisruptionProps {
     inputs: DisruptionInfo;
 }
 
+const getReasonOptions = (): JSX.Element[] => {
+    const options: JSX.Element[] = [];
+
+    options.push(
+        <option value="" key="">
+            Select Reason
+        </option>,
+    );
```

Comment:
```suggestion
    const options: JSX.Element[] = [ <option value="" key="">
            Select Reason
        </option>];
```

We can just start the array with the value inside it

Reply:
done